lib/src/common/disposable.dart

```diff
@@ -112,6 +112,14 @@ class _ObservableTimer implements Timer {
 
   @override
   bool get isActive => _timer.isActive;
+
+  @override
+  int get tick {
+    // TODO: Once fully transitioned to Dart 2 just return
+    // the tick value from our internal timer
+    // return _timer.tick;
+    throw new UnsupportedError('Timer.tick is unsupported');
```

Comment:
Won't this cause this not to work in dart 2 anyhow? Would it be possibly to try to access `_timer.tick` and throw if it throws?

Reply:
Tried that. It is not possible to try to use it if it is there. I asked the Dart team and they suggested just throwing UnsupportedError.